### ISSUE TITLE
how to upload  files using php in lamp ubuntu including settings

### ISSUE THREAD

**Issue by sreekar** [cytopia/devilbox#691]
how to upload  files using php in lamp ubuntu including settings

**Comment by cytopia** [comment]
@sreekar can you rephrase you question and be a bit more specific about what you want to achieve.

**Comment by sreekar** [comment]
actually i am preparing a website using lamp in ubuntu with php  and i have a code to upload file using php  code is not showing any error but its not working

**Comment by cytopia** [comment]
@sreekar If you think this is a bug in Devilbox itself, please do create a bug report ticket  here with steps, files and codes necessary to fully reproduce this. Otherwise people won't really be able to address this.
